src/store/stores/controls.ts

```diff
@@ -127,14 +153,30 @@ export default class ControlStore {
   }
 
   @computed
-  get dataFilters(): Filters {
+  get defaultFilters(): Required<Filters> {
+    return {
+      namespace: null,
+      verdict: null,
+      httpStatus: null,
+      filters: [],
+      skipHost: !this.showHost,
+      skipKubeDns: !this.showKubeDns,
+      skipRemoteNode: !this.showRemoteNode,
+      skipPrometheusApp: !this.showPrometheusApp,
```

Comment:
It looks wrong to have default filters being dependent from current values, lets not do that